lib/inspector.js

```diff
@@ -82,6 +86,37 @@ class Session extends EventEmitter {
   }
 }
 
+// URL construction helper functions.
+function addr() {
+  const id = wsId();
+  if (!id)
+    throw new Error('inspector did not start');
+  return wsHost() + ':' + wsPort() + '/' + id;
+}
+
+function chromeUrl() {
+  return 'chrome-devtools://devtools/bundled/inspector.html'
+       + '?experiments=true&v8only=true'
+       + '&ws=' + addr();
+}
+
+function wsUrl() {
+  return 'ws://' + addr();
+}
+
+
 module.exports = {
+  ws: {
+    id: wsId,
+    listen: wsListen,
+    unlisten: process._debugEnd,
+    // XXX find a way to just stop Io thread, not entire agent, so ID can be stable?
```

Comment:
UUID is there for security, so web sites are not able to connect to Node instances running locally. So it does not represent anything.

It would likely be convinient if it stayed stable over the life of the Node process, but it does not matter much.